assignments/prototypes.js

```diff
@@ -32,6 +53,26 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(attributes) {
+  
+  this.team = attributes.Humanoid;
```

Comment:
looks like there is a little mix up here. Remember what attributes will look like. It's going to be an object with characteristics like:
```
{
createdAt: new Date()
dimensions:{
height:6,
width:4
}
name:'Superman'
team:'Super Friends'
healthPoints:1000
language:'Kryptonese'
weapons:[heat vision, super strength]
}
```
You want to take each of those object attributes and lay them over your object constructor attributes as you did up above with character stats. The reason you are getting an undefined on all of your humanoid properties tests is because you are setting all new object constructor attributes to `attributes.Humanoid` which does not exist in the attributes you pass in. If you were to add `Humanoid:'banana'` to your attributes object, your consoles on team, weapons and language would all  come back as banana.